pkg/recipes/terraform/config/config_test.go

```diff
@@ -666,6 +695,213 @@ func Test_AddOutputs(t *testing.T) {
 	}
 }
 
+func Test_updateModuleWithProviderAliases(t *testing.T) {
```

Comment:
I think we are making the same assumption in test cases that alias names in configuration_alias field have to match the alias names in providers configs, which doesn't seem to be true from official docs.

Reply:
Since we are going ahead with only supporting the scenarios with 1-1 mapping, could you add a unit test for scenario where config_alias has a name that doesn't exist in the provider config?

Reply:
This test exists: ln 775 : "Test with subset of required_provider aliases in provider config". I can add another one where it's not a subset.

Reply:
My suggestion was not for subset - it was for use case 2 discussed yeterday.

"scenario where config_alias has a name that doesn't exist in the provider config"